### ISSUE TITLE
/skin is not working

### ISSUE THREAD

**Issue by glassesglasses8** [kaboomserver/server#145]
/skin from myskin

**Comment by wigfyehkdwi** [comment]
> /skin from myskin

run it until it works

**Comment by shaijyy** [comment]
> command blocks are not repeating

Yes, Command Blocks only runs once, then basically self-destruct. you might want to use *cb via EssentialsX, or write a chain command block chain that automatically resets the command blocks in it

**Comment by amyavi** [comment]
> /skin from myskin

The correct syntax of the command is `/skin <premium player>`.

> command blocks are not repeating

This is an intentional change so that players can't easily make the server unplayable.
